src/shared/Core/HttpClientFactory.cs

```diff
@@ -7,6 +7,7 @@
 using System.Net.Http.Headers;
 using System.Net.Security;
 using System.Security.Cryptography.X509Certificates;
+using CurlCookie;
```

Comment:
Once we name the namespace in the `CurlCookie.cs` file to `GitCredentialManager`, we can drop this `using` statement.

```suggestion
```